packages/appstore/src/components/main-title-bar/asset-summary.tsx

```diff
@@ -41,7 +39,6 @@ const AssetSummary = observer(() => {
     const should_show_loader =
         ((is_switching || is_logging_in) && (eu_account || cr_account)) ||
         !is_landing_company_loaded ||
-        !exchange_rates ||
```

Comment:
are you sure we don't need this? 🤔 
because we added this to prevent showing the user the total balance multiple time while it was still getting calculated 🤔

Reply:
@shayan-deriv tested it a couple of times and it worked fine. But today Matin showed to me one small ui issue when the user goes to another route and comes back, in result the user can see how total asset changes without loader. Will try to add the fix ASAP. Will let you know here when it will be done

Reply:
the rest LGTM 👍